cki-send.py

```diff
@@ -55,12 +43,21 @@ def main(args):
     for (x, y) in instances:
         filename = f"{x}.{y}.result"
         result = pathlib.Path(running_path / filename ).read_text().strip()
-        new_temp = template.copy()
-        new_temp["test_name"] = f"{args.cloud}-{x}"
-        new_temp["test_description"] = f"Kernel test result on instance {x} of {args.cloud}"
-        new_temp["test_arch"] = y
-        new_temp["test_result"] = result
-        msg["results"].append(new_temp)
+        msg["tests"].append({
+            "build_id": args.kcidb_build_id,
+            "id": f'redhat:virt_qe_s1_{args.build_id}_{x}',
+            "origin": "redhat",
+            "environment": {
+                "comment": f'Instance {x} of {args.cloud}'
+            },
+            "path": "ltp"
+            "log_url": f'{args.build_url}/consoleText',
```

Comment:
@henrywang is there a way to get a log file for each test? You mentioned adding files from `report_files`, but for the example job I was looking at it seemed to be an empty list

Reply:
`.result` and `.report` files locate at the same path. https://jenkins-cloudci-prod-virt-qe-3rd.apps.ocp4.prod.psi.redhat.com/job/cki/job/rhel-8.y-gcp/808/artifact/

`reports_files` is a list of file name which does not include path.

Reply:
So it looks like the `*_INSTANCE_TYPE}.${ARCH}.result` file corresponds to the `{{ instance }}-ltp.report` file? Is there a way to associate those in cki-send?

Reply:
Pushed a commit where I add a separate test result for each ltp subtest. Still not sure how to associate the result and the report file - see the comment on line 34